From d54d72cdcbb7459da3f000390932227d925f33a6 Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Fri, 24 Mar 2017 07:23:49 -0500
Subject: [PATCH 2/3] Fix R-package PKGROOT

It pointed to the wrong directory on Windows and had a
trailing / on all systems leading to double /
---
 R-package/src/Makevars.in  | 2 +-
 R-package/src/Makevars.win | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/R-package/src/Makevars.in b/R-package/src/Makevars.in
index 26aa5386..d455ed6e 100644
--- a/R-package/src/Makevars.in
+++ b/R-package/src/Makevars.in
@@ -1,5 +1,5 @@
 # package root
-PKGROOT=../../
+PKGROOT=../..
 ENABLE_STD_THREAD=1
 # _*_ mode: Makefile; _*_
 
diff --git a/R-package/src/Makevars.win b/R-package/src/Makevars.win
index 953b5dc2..4a8fa38b 100644
--- a/R-package/src/Makevars.win
+++ b/R-package/src/Makevars.win
@@ -1,5 +1,5 @@
 # package root
-PKGROOT=./
+PKGROOT=../..
 ENABLE_STD_THREAD=0
 # _*_ mode: Makefile; _*_
 
-- 
2.14.3 (Apple Git-98)

